manifests/w/WiseCleaner/WiseNote/1.0.2/WiseCleaner.WiseNote.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created with komac v2.12.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: WiseCleaner.WiseNote
+PackageVersion: 1.0.2
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: Wise Note_is1
+ReleaseDate: 2025-07-04
+AppsAndFeaturesEntries:
+- Publisher: Lespeed Technology Co., Ltd.
+  ProductCode: Wise Note_is1
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\Wise\Wise Note'
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
----
This errors out on x86
```
Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Program Files\Wise\Wise Note\wisenote.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)
```